### DISCUSSION TITLE
PROJECT_LICENSE

### DISCUSSION THREAD

**Discussion by zellx3** [genyleap/Project-Template#3] [General]
I believe project license should not be as a variable in cmake it must be hard coded, what will happen if I change in build time license to GPL but inside the cmake there is MIT default license.

**Comment by thecompez** [comment]
> I believe project license should not be as a variable in cmake it must be hard coded, what will happen if I change in build time license to GPL but inside the cmake there is MIT default license.

Of course, that variable is not related to the template license, you mentioned a good point. We can have the best solution for it. When your project license is GPL, you must change that variable to GPL.
